### PR TITLE
Update CICD for new armadaproject GH org.

### PR DESCRIPTION
**Pull request by richscott**
Update CICD configs for new armadaproject GH org.

THIS PR WILL NOT BE MERGED UNTIL IMMEDIATELY BEFORE THE REPO-ORG MIGRATION.

### CODE REVIEW

File: docs/dev/docker-compose.yaml [hunk 1]
```diff
@@ -28,8 +28,9 @@ services:
       - "4223:4223"
       - "8223:8223"
   pulsar:
-    image: apachepulsar/pulsar:2.10.0
-    profiles: ["linux"]
+    image: kezhenxu94/pulsar
```
Comment (dave-gantenbein) [inline docs/dev/docker-compose.yaml:31]: Why the change? also, can we remove the commented code?

File: docs/dev/docker-compose.yaml [hunk 2]
```diff
@@ -28,8 +28,9 @@ services:
...
+    # image: apachepulsar/pulsar:2.10.0
```
Comment (kannon92) [inline docs/dev/docker-compose.yaml:32]: Not sure we want this change?

**Review by kannon92 — DISMISSED** [review]
LGTM other than the pulsar config.

File: docs/dev/docker-compose.yaml (hunk 1 above)
Comment (richscott) [reply docs/dev/docker-compose.yaml:31]: UGH! This absolutely was not meant to be included in this diff (heat of the moment hacking here). I"ll remove it.

Comment (richscott) [reply docs/dev/docker-compose.yaml]: This change excised in commit 91f5ccb6

**Review by kannon92 — DISMISSED** [review]
